src/patternfly/layouts/Page/page.scss

```diff
@@ -139,10 +144,14 @@
 .pf-l-page__header-brand-toggle {
   .pf-c-button {
     padding: var(--pf-l-page__header-sidebar-toggle--Padding);
-    margin-right: var(--pf-l-page__header-brand-link--MarginLeft);
-    margin-left: calc(var(--pf-l-page__header-sidebar-toggle--PaddingLeft) * -1); // ensure button does not adjust positioning, offset padding-left
+    margin-right: var(--pf-l-page__header-sidebar-toggle--MarginRight);
```

Comment:
Can you update the `--pf-l-page__header-sidebar-toggle*` vars to `pf-l-page__header-brand-toggle__c-button`?